packages/netlify-cms-core/src/actions/deploys.js

```diff
@@ -1,86 +0,0 @@
-import { actions as notifActions } from 'redux-notifications';
-import { currentBackend } from 'coreSrc/backend';
-import { selectDeployPreview } from 'Reducers';
-
-const { notifSend } = notifActions;
-
-export const DEPLOY_PREVIEW_REQUEST = 'DEPLOY_PREVIEW_REQUEST';
-export const DEPLOY_PREVIEW_SUCCESS = 'DEPLOY_PREVIEW_SUCCESS';
-export const DEPLOY_PREVIEW_FAILURE = 'DEPLOY_PREVIEW_FAILURE';
-
-export function deployPreviewLoading(collection, slug) {
-  return {
-    type: DEPLOY_PREVIEW_REQUEST,
-    payload: {
-      collection: collection.get('name'),
-      slug,
-    },
-  };
-}
-
-export function deployPreviewLoaded(collection, slug, { url, status }) {
-  return {
-    type: DEPLOY_PREVIEW_SUCCESS,
-    payload: {
-      collection: collection.get('name'),
-      slug,
-      url,
-      status,
-    },
-  };
-}
-
-export function deployPreviewError(collection, slug) {
-  return {
-    type: DEPLOY_PREVIEW_FAILURE,
-    payload: {
-      collection: collection.get('name'),
-      slug,
-    },
-  };
-}
-
-/**
- * Requests a deploy preview object from the registered backend.
- */
-export function loadDeployPreview(collection, slug, entry, published, opts) {
-  return async (dispatch, getState) => {
-    const state = getState();
-    const backend = currentBackend(state.config);
-
-    // Exit if currently fetching
-    const deployState = selectDeployPreview(state, collection, slug);
```

Comment:
I pretty sure this was a bug. selectDeployPreview function doesn't accept collection object.

Reply:
Definitely a bug coming from inconsistent naming (`collection` vs `collectionName`).
Not the first time I've seen it happen in the codebase. I'm glad you fixed it